docs/src/app/components/pages/discover-more/Showcase.js

```diff
@@ -106,21 +105,25 @@ const Showcase = () => (
     <Title render={(previousTitle) => `Showcase - ${previousTitle}`} />
     <MarkdownElement text={showcaseText} />
     <GridList
-      cols={4}
+      cols={3}
       cellHeight={200}
       style={styles.gridList}
     >
       {appList.map((app) => (
         <GridTile
           key={app.title}
-          containerElement="a"
-          href={app.link}
-          target="_blank"
+          containerElement="div"
           title={app.title}
           subtitle={<span>{'by '}<b>{app.author}</b></span>}
           style={styles.gridTile}
```

Comment:
Sorry, we are not here yet 😆 . That pointer style is confusing. The whole card isn't interactive.